BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/Location.java

```diff
@@ -112,7 +156,7 @@ public static double getRotationAngleInDegrees(Location centerLocation, Location
                 - (Math.sin(centerLocation.latitude) * Math.cos(targetLocation.latitude) * Math.cos(longitudeDelta));
         double y = Math.sin(longitudeDelta) * Math.cos(targetLocation.latitude);
         double angle = Math.toDegrees(Math.atan2(y, x));
-        return 360 - ((angle + 360) % 360);
+        return (360 - angle) % 360;
```

Comment:
As `angle` can be negative the former formula (`return 360 - ((angle + 360) % 360);`) could still be used if we change it to `return 359 - ((angle + 360) % 360);`. The second part of the formula results to [0, 359] therefore changing `360` - `359` will correct the interval from `(0, 360]` to `[0, 360)`.